src/atoms/SegmentControl/SegmentControl.native.js

```diff
@@ -0,0 +1,94 @@
+import React, { useState, useCallback, useMemo } from 'react';
+import PropTypes from 'prop-types';
+import SegmentControlContext from './SegmentControlContext';
```

Comment:
Move relative imports for `SegmentControlContext` and `SegmentControlOption` below.